source/openapi3/split.yaml

```diff
@@ -2880,175 +2642,46 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ListOutgoingPaymentRequestsResponse'
-  '/payment_requests/{payment_request_ref}/history':
-    get:
+  '/credits/{credit_ref}/refunds':
+    post:
       tags:
-        - Payment Requests
-      summary: Get a Payment Request's history
-      description: >-
-        Gives you visibility of the entire Payment Request lifecycle including
-        the generated debit and credit transactions.
-      operationId: GetAPaymentRequest'sHistory
+        - Refunds
+      summary: Issue a Refund
+      description: >
+        Certain rules apply to the issuance of a refund:
+
+        <ul>
+          <li>Must be applied against a successfully cleared Payment Request (credit)</li>
+          <li>Many refunds may be created against the original Payment Request. The total refunded amount must not exceed the original value</li>
+        </ul>
+      operationId: IssueARefund
       parameters:
-        - name: payment_request_ref
+        - name: credit_ref
           in: path
-          description: 'Single value, exact match'
+          description: 'The credit reference number e.g C.625v'
```

Comment:
Is the trailing `v` in the ref intentional?

Reply:
Yep, was trying to modernize the new descriptions but its a whole other activity and will make these more random and longer next PR